### PR TITLE
[leone] Leone upgrade

### PR DESCRIPTION
**Pull request by lucamar**
Updating filename for the production list of `Leone`.

### CODE REVIEW

**Comment by lucamar** [comment]
```
ERROR: Error cloning remote repo 'origin'

hudson.plugins.git.GitException: Command "git fetch --tags --progress https://github.com/eth-cscs/production.git +refs/heads/*:refs/remotes/origin/*" returned status code 128:

stdout: 

stderr: error:  while accessing https://github.com/eth-cscs/production.git/info/refs
```

File: jenkins-builds/leone-RHEL6.9EUS-17.06 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+ Boost-1.61.0-foss-2016b-Python-2.7.12.eb
+ ddt-18.0.1.eb
```
Comment (gppezzi) [inline jenkins-builds/leone-RHEL6.9EUS-17.06]: one version of ddt should be enough?

Comment (jgphpc) [reply jenkins-builds/leone-RHEL6.9EUS-17.06]: would be the right time to update the recipe (and not use /apps/common)

File: jenkins-builds/leone-RHEL6.9EUS-17.06 [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+ ddt-18.1.1.eb                              --set-default-module
+ h-yade-2017.01a-foss-2016b-Python-2.7.12.eb
```
Comment (gppezzi) [inline jenkins-builds/leone-RHEL6.9EUS-17.06]: The user is currently trying to rebuild yade on his own with EB (they need to re-compile it frequently and having a static version maybe is not so helpful).

For the moment we don't need to include this on the list, if the user does not succeed @ChristopherBignamini will ask to add it back to the list.

File: jenkins-builds/leone-RHEL6.9EUS-17.06 [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+ LIGGGHTS-4.0.eb                            --set-default-module
+ OpenFOAM-4.1-foss-2016b.eb                 --set-default-module
```
Comment (gppezzi) [inline jenkins-builds/leone-RHEL6.9EUS-17.06]: Only latest should be enough for starting? any thoughts @kraushm ?

File: jenkins-builds/leone-RHEL6.9EUS-17.06 (hunk 1 above)
Comment (gppezzi) [reply jenkins-builds/leone-RHEL6.9EUS-17.06]: 👍

Comment (jgphpc) [reply jenkins-builds/leone-RHEL6.9EUS-17.06]: https://github.com/eth-cscs/production/pull/832

**Comment by lucamar** [comment]
See CSCS ticket # 33098

**Comment by lucamar** [comment]
retest this please

File: jenkins-builds/leone-RHEL6.9EUS-17.06 (hunk 3 above)
Comment (kraushm) [reply jenkins-builds/leone-RHEL6.9EUS-17.06]: Anton Kidess (Hilti) replied on 25.09.2018, that they agree to removing the old version OpenFOAM/4.1 from the production list.

**Comment by lucamar** [comment]
I'm closing this pull request, since @kraushm has created an updated branch in his fork.

**Pull request closed without merging**